extensions/amp-onetap/amp-onetap.md

```diff
@@ -0,0 +1,80 @@
+<!--
+  1. Change "category" below to one of:
+       ads-analytics
+       dynamic-content
+       layout
+       media
+       presentation
+       social
+
+  2. Remove any of the "formats" that don't apply.
+     You can also add the "ads" and "stories" formats if they apply.
+
+  3. And remove this comment! (no empty lines before "---")
+-->
+---
+$category: presentation
+formats:
+  - websites
+  - email
+teaser:
+  text: FILL THIS IN.
+---
+<!--
+Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# `amp-onetap`
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>FILL THIS IN</td>
```

Comment:
Maybe we can have a brief explanation for each of the "FILL THIS IN" placeholders?